compiler/src/iree/compiler/Codegen/Common/CPU/test/llvmcpu_materialize_encoding.mlir

```diff
@@ -1,4 +1,4 @@
-// RUN: iree-opt --iree-codegen-cpu-materialize-encoding --canonicalize --cse --split-input-file %s | FileCheck %s
+// RUN: iree-opt --pass-pipeline="builtin.module(func.func(iree-codegen-cpu-materialize-encoding),canonicalize,cse)" --split-input-file %s | FileCheck %s
```

Comment:
This is kind of unfortunate. It's nice to not have to write out the full pass pipeline syntax, but I guess whatever does the automatic nesting isn't smart enough. Does this work with `MultiOpNest`?